src/glyph/mod.rs

```diff
@@ -251,6 +252,49 @@ impl Contour {
     fn is_closed(&self) -> bool {
         self.points.first().map_or(true, |v| v.typ != PointType::Move)
     }
+
+    /// Converts the contour to a kurbo::BezPath
+    #[cfg(feature = "kurbo")]
+    pub fn to_kurbo(&self) -> Result<kurbo::BezPath, ErrorKind> {
+        let mut path = kurbo::BezPath::new();
+        let mut offs: VecDeque<kurbo::Point> = VecDeque::new();
+        // Add end-of-contour offcurves to queue
+        let mut i = self.points.len() - 1;
+        while i > 0 && self.points[i].typ == PointType::OffCurve {
+            let kurbo_point = kurbo::Point::new(self.points[i].x as f64, self.points[i].y as f64);
+            offs.push_front(kurbo_point);
+            i -= 1;
+        }
+        if self.is_closed() {
+            path.move_to(kurbo::Point::new(self.points[i].x as f64, self.points[i].y as f64));
+        }
+        for pt in &self.points {
+            let kurbo_point = kurbo::Point::new(pt.x as f64, pt.y as f64);
+            match pt.typ {
+                PointType::Move => path.move_to(kurbo_point),
+                PointType::Line => path.line_to(kurbo_point),
+                PointType::OffCurve => offs.push_back(kurbo_point),
+                PointType::Curve => {
+                    match offs.len() {
+                        0 => return Err(ErrorKind::BadPoint),
+                        1 => path.quad_to(offs[0], kurbo_point),
+                        2 => path.curve_to(offs[0], offs[1], kurbo_point),
+                        _ => return Err(ErrorKind::TooManyOffCurves),
+                    };
+                    offs.truncate(0);
+                }
+                PointType::QCurve => {
+                    while offs.len() > 1 {
+                        let implied_point = offs[0].midpoint(offs[1]);
+                        path.quad_to(offs.pop_front().unwrap(), implied_point);
+                    }
+                    path.quad_to(offs[0], kurbo_point);
+                    offs.truncate(0);
```

Comment:
again I've developed an allergy to indexing, and would personally probably do something like:

```rust
while let Some(pt) = offs.pop_front() {
    if let Some(next) = offs.front() {
        let implied_point = pt.midpoint(*next);
        path.quad_to(pt, implied_point);
    } else {
        path.quad_to(pt, kurbo_point);
    }
}
```